src/App/Components.fs

```diff
@@ -8,17 +8,21 @@ module Color =
     let context = Solid.createContext()
 
 type Components =
+
     [<JSX.Component>]
     static member ColorProvider(color: string, children) =
         let color, setColor = Solid.createSignal(color)
         Color.context.Provider((color, setColor), children)
 
-    [<JSX.Component>]
-    static member Div (classes: string seq) children =
-        Html.div [
-            Attr.classList classes
-            Html.children children
-        ]
+
+    // does not work anymore(2024), generates invalid JSX: <Components_Div classes={["column", "is-4"]}>
```

Comment:
I am not sure why, but this component compiles to invalid JSX, `<Components_Div classes={["column", "is-4"]}>`